### DISCUSSION TITLE
Model quality in webXR vs scene viewer

### DISCUSSION THREAD

**Discussion by FutureEyes** [google/model-viewer#3826] [Q&A]
### Description

the model when load with webXR is more detail and not blurry like in scene viewer, any chance to make it have the same quality for both webXR and scene-viewer ?

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: vX.X.X

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Can you post some comparative screenshots or recordings so we can see what you mean? Also what kind of device? Is there a reason you prefer not to use WebXR? It's our default option because it's faster, the rendering is consistent, and it's more customizable.

**Reply by vortice3D** [reply]
Hi there, @elalish and @FutureEyes :

I was googling about this same subject and, well, as this thread is not closed yet, I suppose I can invite myself to the conversation.

The case is that I hoped model-viewer AR mode to be faster than webxr mode, as indeed I remenber it was that way, in previous versions (I can't tell now which specific ones); but now seems not to be the case.

In general, scene-viewer mode lasts a lot in showing even a less-than-1MB GLB file. In the meanwhile you can see the blue spinner twirling in an endless loop. You think allí the thing is halted or hung. Anyway, once the model is finally shown, the experience is more robust than with its webxr mode counterpart. Framerate is better and, very important for users, you can take photos and videos. Also, it plants the 3D model in  place and scale.

The webxr mode shows the 3D model much more quickly, but not in correct scale, and after a few seconds it jumps to other posición, where after another pair of seconds is then enlarged to its correct size. The framerate and the experience is more clearly choppier.

Al this, obviously, tested on the same devices. In my case a Moto G8, a Samsung Tab S5 and a Xiaomi Redme Note 8T.

Going to iOS, using an iPad 10" (8th Gen), even with the on-the-fly conversión to USDZ (terrific functionality here) the AR experience is much more satisfactory. Indeed, that might be the target for both webxr and scene-viewer on Android.

Best regards.
